src/lib/pages/backoffice/Item/ItemDetails/ItemActionMenu/ItemActionMenu.js

```diff
@@ -164,9 +180,11 @@ ItemActionMenu.propTypes = {
   deleteItem: PropTypes.func.isRequired,
   checkoutItem: PropTypes.func.isRequired,
   checkOutitemList: PropTypes.arrayOf(PropTypes.object),
+  isInCirculation: PropTypes.bool,
 };
 
 ItemActionMenu.defaultProps = {
   offset: 0,
   checkOutitemList: [],
+  isInCirculation: false,
```

Comment:
I would go maybe for something more explicit?
```suggestion
  isCurrentlyOnLoan: false,
```